components/AlertModal.tsx

```diff
@@ -0,0 +1,60 @@
+import { Button } from '@chakra-ui/button';
+import { WarningTwoIcon } from '@chakra-ui/icons';
+import { Flex, Text } from '@chakra-ui/layout';
+import {
+  Modal,
+  ModalBody,
+  ModalCloseButton,
+  ModalContent,
+  ModalFooter,
+  ModalHeader,
+  ModalOverlay,
+} from '@chakra-ui/modal';
+import { Address } from 'abitype';
+
+type AlertModalProps = {
+  isOpen: boolean;
+  onClose: () => void;
+  onClick: () => void;
+  address: Address;
+  description: string;
+  children?: React.ReactNode;
+};
+
+const AlertModal = (props: AlertModalProps) => {
+  return (
+    <Modal blockScrollOnMount={false} isOpen={props.isOpen} onClose={props.onClose} isCentered>
+      <ModalOverlay />
+      <ModalContent>
+        <ModalHeader></ModalHeader>
+        <ModalCloseButton />
+        <ModalBody>
+          <Flex direction="column">
+            <Flex justify="center" mb="1rem" color="red.500">
+              <Flex bg="red.100" px=".50rem" py=".25rem" alignItems="center" borderRadius=".75rem">
```

Comment:
I think we should use a yellow color instead here since this is a warning, and red usually means error